cmd/nse/nse.go

```diff
@@ -161,3 +260,18 @@ func socketCleanup(listenEndpoint string) error {
 	}
 	return nil
 }
+
+func getCurrentNS() string {
+	buf := make([]byte, MaxSymLink)
+	numBytes, err := syscall.Readlink(netnsfile, buf)
+	if err != nil {
+		return ""
+	}
+	link := string(buf[0:numBytes])
+	nsRegExp := regexp.MustCompile("net:\\[(.*)\\]")
+	submatches := nsRegExp.FindStringSubmatch(link)
+	if len(submatches) >= 1 {
+		return submatches[1]
+	}
+	return ""
```

Comment:
Should we return an empty string here, or return an error?

Reply:
I have no strong opinion here, if you feel this func signature should be `(string, error)`, I can change it.